### PR TITLE
homebrew modules: PEP8 compliancy and documentation changes

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This PR includes:
- PEP8 compliancy changes
- Documentation changes

##### ISSUE TYPE
 - Feature Pull Request
 - Docs Pull Request

##### COMPONENT NAME
homebrew en homebrew_cask

##### ANSIBLE VERSION
v2.5

### CODE REVIEW

**Comment by danieljaouen** [comment]
While I'm all for the documentation changes, I'm not sure about the pep8 changes.  Doesn't pep8 warn about long lines?  I'd like to keep line length 72 characters and below, if possible.

**Comment by dagwieers** [comment]
@danieljaouen The standard PEP8 line length is 79, however the Ansible project's opinion (like many other projects) is that everyone has a large screen and this limitation is no longer valid. So the line length was increased to 159 characters.

PEP8 is mostly about having readable code, and having to split lines at 79 characters is seldom convenient and does not necessarily help readability. But this is a subjective topic and mostly conforms to what people are used to. Implementing something like PEP8 means adjusting your personal customs to a common standard.

Can you please comment per line what you'd prefer to see different ?

File: lib/ansible/modules/packaging/os/homebrew.py [hunk 1]
```diff
@@ -261,17 +242,7 @@ def valid_state(cls, state):
...
-                and state.lower() in (
-                    'installed',
-                    'upgraded',
-                    'head',
-                    'linked',
-                    'unlinked',
-                    'absent',
-                )
-            )
+            return isinstance(state, string_types) and state.lower() in ('absent', 'head', 'installed', 'linked', 'unlinked', 'upgraded')
```
Comment (danieljaouen) [inline lib/ansible/modules/packaging/os/homebrew.py:245]: For example, this line.

**Comment by dagwieers** [comment]
I am going to close this. Feel free to implement the changes you need.

**Pull request closed without merging**